agent/brandon/search.py

```diff
@@ -21,8 +25,25 @@ def _estimate_move_score(board, move):
         return (len(move.get_cells())
             + WEIGHT_SUMITO * move.is_sumito(board))
 
+    @staticmethod
+    def _is_quiescent(board):
+        """
+        Determines if the given board has adjacent opposing marbles.
+        :param board: a Board
+        """
+        for cell, color in board.enumerate_nonempty():
+            for neighbor in cell.neighbors_se():
+                neighbor_color = board[neighbor] if neighbor in board else None
+                if neighbor_color and neighbor_color != color:
+                    return False
+        return True
```

Comment:
There's definitely a faster way to do this with DP a la algos lab but I'm too tired atm to think up something for that lol